### DISCUSSION TITLE
use js sdk to listen transfer event error

### DISCUSSION THREAD

**Discussion by shaobinyao** [zkSync-Community-Hub/zksync-developers#78] [SDKs] [answered]
### Environment

Mainnet

### Acknowledgement

- [ ] I understand that issues are expected with the Python and Java SDK as they are being updated.

### Issue Description

{
  jsonrpc: '2.0',
  error: { code: -32600, message: 'Invalid request' },
  id: null
}

### Expected Behavior

return transfer event

### Code Example

const provider = await zksync.getDefaultProvider('mainnet');
  // Connect to the event server.
  const ws = new WebSocket('wss://mainnet.era.zksync.io/ws');  //wss://events.zksync.io/  wss://goerli-events.zksync.io/  wss://mainnet.era.zksync.io/ws
  console.log('Connection established');
  // Change the address to the account you're intrested in.
  const ACCOUNT_ADDRESS = '0x**********';

  // Once connected, start sending ping frames.
  setInterval(function () {
    ws.ping();
  }, 10000);

  // Register filters.
  ws.on('open', function open() {
    console.log('open...')
    ws.send(JSON.stringify({
      "account": {
        "status": "committed",
        "accounts": [ACCOUNT_ADDRESS]
      }
    }));
  });

**Comment by dutterbutter** [comment]
@shaobinyao can you provide insights into the code provided? Did you see an example you were trying to replicate? What request are you attempting to make?

**Reply by AlbionaHoti** [reply]
Hi @shaobinyao, this is a follow up for the question @dutterbutter butter asked.

**Reply by idea404** [reply, marked as answer]
Also, @shaobinyao please specify what you expect. The code you provided does not need the line where `provider` is declared. Running the following creates the web socket:

```js
const WebSocket = require("ws");

// Connect to the event server.
const ws = new WebSocket("wss://mainnet.era.zksync.io/ws"); //wss://events.zksync.io/ wss://testnet.era.zksync.dev/ws wss://mainnet.era.zksync.io/ws
console.log("Connection established");
// Change the address to the account you're intrested in.
const ACCOUNT_ADDRESS = "0x**********";

// Once connected, start sending ping frames.
setInterval(function () {
  ws.ping();
}, 10000);

// Register filters.
ws.on("open", function open() {
  console.log("open...");
  ws.send(
    JSON.stringify({
      account: {
        status: "committed",
        accounts: [ACCOUNT_ADDRESS],
      },
    })
  );
});
```

**Comment by idea404** [comment]
Closing discussion as left without reply for 2 weeks
